SConstruct

```diff
@@ -2490,6 +2502,10 @@ def doConfigure(myenv):
         # http://lists.llvm.org/pipermail/cfe-dev/2015-November/045852.html
         #
         if env.TargetOSIs('posix') and not env.ToolchainIs('clang') and conf.CheckForFortify():
+            if conf.CheckForFortifyDefined():
+                conf.env.Prepend(
+                    CPPFLAGS='-U_FORTIFY_SOURCE'
```

Comment:
Having thought about this a bit more, this really feels like a bug in the packaging of the compiler on this system. I'm fine with the distro toolchain wanting to default _FORTIFY_SOURCE=1, but it shouldn't cause a compilation error if I then chose to set _FORTIFY_SOURCE=2 on the command line, as I'm entirely entitled to do. That should simply cause the compiler to honor my choice rather than its default, not raise an error. While I'd be willing to pull this PR as a workaround, I think it might be better to try to first raise this as an issue with your distro toolchain maintainer.